src/Symfony/Component/Mailer/Tests/Transport/Smtp/EsmtpTransportFactoryTest.php

```diff
@@ -109,6 +109,23 @@ public static function createProvider(): iterable
             $transport,
         ];
 
+        $transport = new EsmtpTransport('example.com', 465, true, null, $logger);
+        /** @var SocketStream $stream */
+        $stream = $transport->getStream();
+        $streamOptions = $stream->getStreamOptions();
+        $streamOptions['ssl']['peer_fingerprint'] = '6A1CF3B08D175A284C30BC10DE19162307C7286E';
```

Comment:
Does it make sense to name it `verify_peer_fingerprint` like `verify_peer_name` is named?

Reply:
I think aligning with PHP options name would give a better idea what it actually does. Also `verify_*` sounds like a boolean, but we need to specify the fingerprint.